client/src/components/Reviews/PhotoModal.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
```

Comment:
I understand why you did this, but since there's a lot of duplicated code here, we should probably work on reducing that duplication at some point.